kalite/coachreports/models.py

```diff
@@ -267,31 +260,31 @@ def user_progress_detail(cls, user_id, playlist_id):
                         "path": leaf_node["path"],
                     }
 
-            elif kind == "Quiz":
-                entity_id = playlist["id"]
-                if quiz_log:
-                    if quiz_log.complete:
-                        if quiz_pct_score <= 59:
-                            status = "fail"
-                        elif quiz_pct_score <= 79:
-                            status = "borderline"
-                        else:
-                            status = "pass"
-                    elif quiz_log.attempts:
-                        status = "inprogress"
-                    else:
-                        status = "notstarted"
-
-                    quiz_log_id = quiz_log.quiz
-
-                    entry = {
-                        "id": quiz_log_id,
-                        "kind": "Quiz",
-                        "status": status,
-                        "score": quiz_pct_score,
-                        "title": playlist.get("title"),
-                        "path": "",
-                    }
+            # elif kind == "Quiz":
```

Comment:
I'll take your word that these changes are okay, but can you either remove the commented out code or add some context as to why it remains there?